main.rb

```diff
@@ -0,0 +1,87 @@
+require_relative 'app'
+
+def print_menu
+  puts 'Please choose an option:'
+  puts '1. List all books'
+  puts '2. Add a book'
+  puts '3. Rent a book'
+  puts '4. Return a book'
+  puts '5. List all rentals'
+  puts '6. Quit'
+end
+
+def add_book(library)
+  puts 'Enter the title of the book:'
+  title = gets.chomp
+  puts 'Enter the author of the book:'
+  author = gets.chomp
+  puts 'Enter the publication date of the book:'
+  date = gets.chomp
```

Comment:
- [ ] When creating a book, only the title and author should be requested. We do not need the publication date.